src/styles/abstracts/_variables.scss

```diff
@@ -6,6 +6,9 @@ $light: #fff;
 $dark: #000;
 $shuttle-gray: #5a6570;
 $picton-blue: #47a1f5;
+$royal-blue: #5358d8;
+$cornflower-blue: #638cff;
+$signup-button-color: rgba(0, 0, 0, 0.16);
```

Comment:
Correct me if I'm wrong, Do we need to keep it as the same unit as above?

Reply:
In my knowledge rbg**a** (alpha) mean to add an alpha variable which is an opacity to your color. In this case opacity value for ``signup-button-color`` is 16% which I don't think I can convert rbga into hex-color. If I'm wrong, Can you show your ways or any references to share? 😁